index.js

```diff
@@ -1,20 +1,44 @@
-let { readFile, stat } = require("fs").promises;
-let path = require("path");
-let { FileFinder } = require("faucet-pipeline-core/lib/util/files/finder");
+let { readFile, stat } = require("node:fs/promises");
+let path = require("node:path");
+let { FileFinder } = require("./lib.js");
+/** @import {
+  *   Config,
+	*   AssetManager,
+	*   FaucetPlugin,
+	*   FaucetPluginOptions,
+	*   FaucetPluginFunc,
+	*   CompactorMap,
+	*   WriteFileOpts,
+	*   ProcessFile
+	*   } from "./types.ts"
+	**/
 
 module.exports = {
 	key: "static",
 	bucket: "static",
 	plugin: faucetStatic
 };
 
+/**
+ * The static plugin copies files with an option to define compaction functions
+ *
+ * @type FaucetPlugin<Config>
+ */
 function faucetStatic(config, assetManager, { compact } = {}) {
 	let copiers = config.map(copyConfig =>
 		makeCopier(copyConfig, assetManager, { compact }));
 
 	return filepaths => Promise.all(copiers.map(copy => copy(filepaths)));
 }
 
+/**
+ * Create a copier for a single configuration
```

Comment:
FWIW, I'd prefer descriptive grammar here: "Create_s_" (same below, of course) - YMMV